app/views/aleph/full_item_status.html.erb

```diff
@@ -2,49 +2,68 @@
 <ul class="list-local-locations">
 <% @status.each do |s| %>
   <li class="result-local-location">
-    <%= s[:library] %> <%= s[:collection] %>: <%= s[:call_number] %>
-    <% if s[:description].present? %> <%= s[:description] %> <% end %>
-    <span class="sr"><%= s[:label] %></span>
+    <div class="location-wrap">
+      <span class="location">
+        <span class="library"><%= s[:library] %></span>
+        <span class="collection"><%= s[:collection] %>:</span>
+        <span class="callno"><%= s[:call_number] %></span>
+        <% if s[:description].present? %> <%= s[:description] %> <% end %> <!-- unknown -->
+        <span class="sr"><%= s[:label] %></span> <!-- unknown -->
+      </span>
+      <span class="location-actions">
+        <a href="<%= map_link(s[:library]) %>"
+           class="btn button-subtle map-link">
+           <i class="fa fa-map-marker" aria-hidden="true"></i>
+           <span class="sr">View <%= s[:library] %> location map</span>
+        </a>
+        
+        <a href="" class="btn button-subtle text-callno"><i class="fa fa-mobile-phone" aria-hidden="true"></i> <span class="sr">Text call number</span></a> <!-- not implemented? -->
```

Comment:
We aren't going to get texting in for this initial work. Can we pull this out now and open a backlog ticket to consider adding that feature?

Reply:
yes - i will remove.